docs/ref/databases.txt

```diff
@@ -40,6 +40,10 @@ database connection at the end of each request. To enable persistent
 connections, set :setting:`CONN_MAX_AGE` to a positive integer of seconds. For
 unlimited persistent connections, set it to ``None``.
 
+Persistent connections should be disabled when using ASGI. You can use your
```

Comment:
Thought: Is it worth expanding this to mention the symptoms of it being misconfigured?

> Persistent connections should be disabled when using ASGI, to prevent running out of database connections.

Perhaps also including something about _why_ that's the case (something something concurrency).

Reply:
Maybe...

```suggestion
Due to the way threads are handled, persistent connections should be disabled when using ASGI. You can use your
```

Reply:
That wording might be misleading, since most people will probably ask "But ASGI uses an event loop, not threads".

My reading of the issue is that it's caused by the fact ASGI can handle _many_ requests at once, but doesn't close the connection until the end of the request. Perhaps that should be the focus?

```suggestion
Because it can handle a large number of concurrent requests, persistent connections should be disabled when using ASGI. You can use your
```

~Naming~ Phrasing things is hard!

Reply:
I'm just not sure it's helpful to folks to go into details here...

Either you know, and anything we write is unnecessary, or you don't, and anything we write is just going to muddy the waters more.

🤷